### PR TITLE
chore(clerk-js): Fix transform css rule in outAnimation

### PR DESCRIPTION
**Pull request by kaftarmery**
## Checklist

- [ ] `pnpm test` runs as expected.
- [ ] `pnpm build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Review by alexcarpenter — COMMENTED** [review]
@kaftarmery good catch.

Will need to update the PR title to the format required. e.g. `fix(clerk-js): Fix typo in outAnimation` as an example.

Will also need to generate a changeset, which can be triggered via `npx changeset` in the repo, and follow the steps. https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

File: packages/clerk-js/.changeset/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changesets
+
+Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works
+with multi-package repos, or single-package repos to help you version and publish your code. You can
+find the full documentation for it [in our repository](https://github.com/changesets/changesets)
+
+We have a quick list of common questions to get you started engaging with this project in
+[our documentation](https://github.com/changesets/changesets/blob/main/docs/common-questions.md)
```
Comment (panteliselef) [inline packages/clerk-js/.changeset/README.md:8]: can you revert this and run `pnpm changeset` at the root of the monorepo ?

File: packages/clerk-js/.changeset/config.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "$schema": "https://unpkg.com/@changesets/config@3.0.3/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": []
+}
```
Comment (panteliselef) [inline packages/clerk-js/.changeset/config.json:11]: can you revert this and run pnpm changeset at the root of the monorepo ?

**Review by panteliselef — CHANGES_REQUESTED** [review]

**Review by panteliselef — APPROVED** [review]

File: .changeset/real-fans-fry.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+---
+---
```
Comment (anagstef) [inline .changeset/real-fans-fry.md:2]: We need a changeset here! It's a change that impacts the customers.

Comment (anagstef) [reply .changeset/real-fans-fry.md:2]: It looks like this will not have an impact.

**Pull request merged**